slsim/Pipelines/sl_hammocks_pipeline.py

```diff
@@ -129,7 +215,7 @@ def table_translator_for_slsim(table, cosmo):
     if "ellipticity" not in table.colnames:
         table.rename_column("e_g", "ellipticity")
     if "angular_size" not in table.colnames:
-        angular_size_in_deg = table["tb"] / 0.551 * constants.arcsec
+        angular_size_in_deg = table["tb"] / 0.551
```

Comment:
is this in degree or arc seconds, if the ladder, then perhaps change the variable name